tests/lib/snaps/basic-run/bin/echo

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+echo $1
```

Comment:
Nitpick, use `"$@"` instead of `$1` as that has special expansion rules and lets us use multiple arguments.